cypress/integration/settings/image-picture-check.js

```diff
@@ -0,0 +1,40 @@
+/* global cy */
+
+describe('Picture input field', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains("Get started (it's free)").click({ force: true });
```

Comment:
Can we use the login command to keep consistency?

```suggestion
    cy.login()
```